pos_pricelist/static/src/js/models.js

```diff
@@ -73,60 +73,6 @@ function pos_pricelist_models(instance, module) {
         }
     });
 
-    /**
-     * Extend the order
-     */
-    module.Order = module.Order.extend({
-        /**
-         * override this method to merge lines
-         * TODO : Need some refactoring in the standard POS to Do it better
-         * TODO : from line 73 till 85, we need to move this to another method
-         * @param product
-         * @param options
-         */
-        addProduct: function (product, options) {
-            options = options || {};
-            var attr = JSON.parse(JSON.stringify(product));
-            attr.pos = this.pos;
-            attr.order = this;
-            var line = new module.Orderline({}, {
-                pos: this.pos,
-                order: this,
-                product: product
-            });
-            var self = this;
-            var found = false;
-
-            if (options.quantity !== undefined) {
-                line.set_quantity(options.quantity);
-            }
-            if (options.price !== undefined) {
-                line.set_unit_price(options.price);
```

Comment:
Maybe this is for setting the correct price and it's needed?

Reply:
No it's not. you'll find the same line in the Odoo code. 
see original overwritten function : 
https://github.com/odoo/odoo/blob/8.0/addons/point_of_sale/static/src/js/models.js#L1066

+ 

the comment of the function in the OCA module talks about lines merging.
https://github.com/OCA/pos/pull/236/files/c879f74940fbd11ed3f13d75c820fd500980d3c4#diff-747f62729cafe771e948f5cfeb9aff46L81

regards.